scripts/mlflow_export.py

```diff
@@ -0,0 +1,350 @@
+        results_file: Optional[str] = None,
+    ) -> str:
+        """
+        Export evaluation results to MLFlow based on configuration.
+
+        Args:
+            output_dir: Directory containing evaluation outputs
+            export_types: List of export types ('metrics', 'artifacts')
+            results_file: Specific results file to load (optional)
+
+        Returns:
+            MLFlow run ID
+        """
+        # Find and load results file
+        results_data = None
+        results_file_path = None
+
+        if results_file:
+            provided_path = Path(results_file)
+            if provided_path.exists():
+                results_file_path = str(provided_path)
+            else:
+                print(f"WARNING: Specified results file {results_file} does not exist")
+
+        if results_file_path is None:
+            results_files = sorted(
+                glob.glob(os.path.join(output_dir, "**", "*result*.json"), recursive=True),
+                key=os.path.getmtime,
+                reverse=True,
+            )
+
+            if results_files:
+                results_file_path = results_files[0]
+
+        if results_file_path:
+            try:
+                with open(results_file_path, 'r') as f:
+                    results_data = json.load(f)
+                print(f"Loaded results from {results_file_path}")
+            except Exception as e:
+                print(f"ERROR: Failed to load results from {results_file_path}: {e}")
+        else:
+            print("WARNING: No result files found in output directory")
+
+        # Start MLFlow run
+        try:
+            with mlflow.start_run(run_id=self.run_id, experiment_id=self.experiment_id) as run:
+                run_id = run.info.run_id
+                print(f"Started MLFlow run: {run_id}")
+
+                # Log basic run information
+                mlflow.set_tag("lm_eval.version", "latest")
+                mlflow.set_tag("lm_eval.output_dir", output_dir)
+                if self.source_name:
+                    mlflow.set_tag("mlflow.source.name", self.source_name)
+                if self.source_type:
+                    mlflow.set_tag("mlflow.source.type", self.source_type)
+                if self.params:
+                    # Convert all values to strings for MLflow params
+                    mlflow.log_params(
+                        {k: json.dumps(v) if isinstance(v, (dict, list)) else v for k, v in self.params.items()}
+                    )
+
+                # Export metrics if requested
+                if 'metrics' in export_types and results_data:
+                    print("Exporting metrics to MLFlow...")
+                    self.export_metrics(results_data, run_id)
+
+                # Export artifacts if requested
+                if 'artifacts' in export_types:
+                    print("Exporting artifacts to MLFlow...")
+                    self.export_artifacts(output_dir, run_id)
```

Comment:
I would make a proposal following #59 , to instead set artifact_uri to the OCI reference exported ...wdyt?